File: UnitTestProject/UnitTestProject/Models/Cats.cs
using Supabase.Postgrest.Attributes;
using Supabase.Postgrest.Models;


namespace UnitTestProject.Models
{

    [Table("users")]
    public class Cats : BaseModel
    {
        [PrimaryKey("id")]
        public Guid Id { get; set; }

        [Column("name_cat")]
        public string? NameCat { get; set; }

        [Column("gender_id")]
        public int? IdGender { get; set; }

        [Column("age")]
        public int? Age { get; set; }

        [Column("breed_id")]
        public int? IdBreed { get; set; }

        [Column("weight")]
        public int? Weight { get; set; }

        [Column("description_cats")]
        public string? Description { get; set; }

        [Column("image_url")]
        public string? ImageUrl { get; set; }
    }
}

File: app/src/main/java/com/example/meowmates/model/database/Cats.kt
package com.example.meowmates.model.database

import kotlinx.serialization.Serializable

@Serializable
data class Cats (
    var id: Int =0,
    var name_cat: String = "",
    var gender_id: Int =0,
    var age: Int=0,
    var breed_id: Int=0,
    var weight: Int = 0,
    var description_cats: String ="",
    var image_url: String =""
)